app/library/ng2c3.component.ts

```diff
@@ -74,7 +75,7 @@ export class Ng2C3 {
   private ngOnChanges( changes: { [propertyName: string]: SimpleChange } ): void {
     try {
       this.__render( this.data );
-    } catch(err : Error) {
+    } catch(err) {
```

Comment:
Lemme check more on that. For now I was able to find that closely relates to this, https://github.com/Microsoft/TypeScript/issues/298